posts/inside-rust/2020-03-19-all-hands-retrospective.md

```diff
@@ -0,0 +1,146 @@
+We thought we'd done a good job, though it would turn out that we were quite
+off in our calculations.
+
+Based on our estimated budget, it was clear that Mozilla was not going to be
+able to cover the costs alone. It seemed like a good opportunity to try and
+build relationships with some of the companies that have started to use Rust
+by approaching them for sponsorships. We weren't sure, though, how many
+sponsorships we'd be able to get, and so we hit upon the idea of using
+paid tickets.
+
+The idea was that people could purchase a ticket to the event, which would
+help to cover the costs, but that it was also okay to not have a ticket.
+Tickets have the advantage of being relatively easy for people to submit in
+their expense report if they are travelling with a company. We also found a
+number of companies who might be interested in a more traditional sponsorship.
+
+### Finding a bank account.
+Still, we had another problem. The Rust organisation does not have a bank
+account we could use to collect and distribute money, and we didn't have an
+obvious entity to sign legal agreements.
+
+At this point, we spoke to a company that might serve as a bank account. We
+had a quick call and went over a plan that seemed promising. Mozilla would
+contract with them for their services managing and organising the event, and
+this would serve as Mozilla's sponsorship. The other sponsorships and proceeds
+from ticket sales would collect in their bank account and we would
+manage distributions.
+
+### Pulling the trigger.
+At that point, we sent mail encouraging people to book tickets, and we also
+started following up more with sponsors. Our expectation was that we would
+soon have a bank account for collecting funds, and we started collecting
+information to repay people who had booked travel.
+
+In retrospect, it's clear that this was premature. It seemed like all the
+pieces were in place, and we knew that if people didn't start booking tickets,
+they never would. But the pieces were not in place, not actually, and we clearly
+should have taken this moment ask whether we were really going to be able to
+pull this off.
+
+### Realising it won't work.
+Finally, this all started to unravel in a about a month away from the event.
+In short, we encountered some logistical hurdles, which we won't go into, but
+which also prompted us to review the finances in more detail. This revealed
+that our estimates were off. Thus, we were faced with a tough decision:
+
+* If we continue with the event, there was the chance that the logistics would
+  just fail to come together. That would be a true disaster.
+* Further, we might not be able to raise enough money to be able to pay for
+  the entirety of people's travel and lodging as we had promised. This would be
+  unfortunate, though perhaps something people could live with.
+* On the other hand, if we cancel, we have to deal with the fact that people
+  had already started booking travel.
```

Comment:
add something like "and knowing that it *might* have worked if only we had tried" or similar?